helm/ako/crds/nsxalbinfra.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: nsxalbinfrasettings.ako.vmware.com
```

Comment:
should we rename the file to match the crd name `nsxalbinfrasettings.yaml`

Reply:
Done.